README.md

```diff
@@ -76,12 +76,20 @@ addEventListener("message", async ({data}) => {
 });
 ```
 
+### Worklets
```

Comment:
Editorial nit: I would group Worklets with Workers, and put structured clone as a separate item.